### PR TITLE
Update Bioconductor to version 3.10

### PR DESCRIPTION
**Pull request by nturaga**
This PR will update the version of Bioconductor to the latest release 3.10.

### CODE REVIEW

File: terra-jupyter-bioconductor/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.0.6 - 11/22/2019
+- Use new Bioconductor version 3.10
+- Update `terra-jupyter-bioconductor` version to `0.0.6`
+
+`Image URL: us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:0.0.5`
```
Comment (Qi77Qi) [inline terra-jupyter-bioconductor/CHANGELOG.md]: version should 0.0.6

File: terra-jupyter-bioconductor/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
-FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.6
+#FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.7
+FROM terra-jupyter-r:0.0.7
```
Comment (Qi77Qi) [inline terra-jupyter-bioconductor/Dockerfile]: this needs to be fixed

File: terra-jupyter-bioconductor/CHANGELOG.md (hunk 1 above)
Comment (nturaga) [reply terra-jupyter-bioconductor/CHANGELOG.md]: Done

File: terra-jupyter-bioconductor/Dockerfile (hunk 2 above)
Comment (nturaga) [reply terra-jupyter-bioconductor/Dockerfile]: Done

**Pull request merged**